modules/tinymce/CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Moved non-UI table functionality to core #TINY-8273
 - The `table_style_by_css` option has been set to `true` by default #TINY-8259
 - The `table_use_colgroups` option has been set to `true` by default #TINY-8259
+- Pressing Shift + Enter will now step out of `blockquote` elements to match `pre` element behavior #TINY-6559
```

Comment:
This needs rebasing on top of `next` since it's for 6.1 and this entry will need moving to the 6.1 unreleased section (as this section is for 6.0).

Reply:
I'll do this after the discussion that you requested.